dandi/pynwb_utils.py

```diff
@@ -362,14 +387,24 @@ def validate(path: Union[str, Path], devel_debug: bool = False) -> List[str]:
         if version is not None:
             # Explicitly sanitize so we collect warnings.
             # TODO: later cast into proper ERRORs
+            # Do we really need to do these acrobatics? string comparison works fine...
```

Comment:
what "acrobatics", could you be more specific or this comment is no longer pertinent?

Reply:
done 37bfe870fb4e021cec5b92eaa528e54df25d8871